stdlib/public/TensorFlow/DataTypes.swift

```diff
@@ -37,6 +39,7 @@ public struct _TensorDataType {
 public protocol AccelerableByTensorFlow {
   /// The underlying TensorFlow data type.
   /// - Note: This is not intended for general use.
+  @inlinable
   static var _tensorFlowDataType: _TensorDataType { get }
```

Comment:
1. Since this will be used outside this module, it's better to drop the underscores from both `_tensorFlowDataType` and `_TensorDataType`.

2. Once dropping underscore is finished, we should use `tensorFlowDataType` everywhere in `#tfop`s, instead of having the inconsistency that the TF module uses `cDataType` and others use `tensorFlowDataType`.

Reply:
can you remind me what value does _TensorDataType serve? why don't we use TF_DataType directly for simplicity?

Reply:
It's API packaging to prevent leaking C symbols, which would then screw up code completion (thus user experience). This has been discussed extensively earlier in the project.

Reply:
I'll add a comment explaining this so that future readers know why the wrapper exists.